### DISCUSSION TITLE
Server stops responding and kicks everyone

### DISCUSSION THREAD

**Discussion by hugovdev** [Minestom/Minestom#1534] [Help]
So we were just minding our own business and this happened:
![idea64_8dcWn4WK4P](https://user-images.githubusercontent.com/23460704/203423602-532a4eba-c90f-4cad-81b2-3d9e18e7e873.png)
![idea64_nT5t2RwELh](https://user-images.githubusercontent.com/23460704/203423619-0d6869ed-a985-4b74-bef8-8255fc0299ad.png)

**Comment by hugovdev** [comment]
![image](https://user-images.githubusercontent.com/23460704/203424233-4861c36b-e647-489d-8faf-f92e8c9592f4.png)
Line 103 is causing the problem, any ideas why?

**Reply by Timofffee** [reply]
Can I have more information? What kind of client? What were you doing at that moment?

It just might happen that the client sent some strange packet that the server failed to process. Although it shouldn't have broken the server.

**Reply by hugovdev** [reply]
There were two people online, I'm using a client with mods but it is very basic:
![image](https://user-images.githubusercontent.com/23460704/204013052-56185bfe-525d-455d-8ec7-2dcc33f7e534.png)
I don't think we were doing anything special just wandering/flying around.
